docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -71,6 +93,57 @@ Create a Data Asset to define the data you want GX Cloud to access. Currently, t
 
 12. Create an Expectation. See [Create an Expectation](/cloud/expectations/manage_expectations.md#create-an-expectation).
 
+</TabItem>
+<TabItem value="PostgreSQL">
+
+Define the data you want GX Cloud to access within PostgreSQL.
+
+### Prerequisites
+
+- You have a PostgreSQL database, schema, and table.
+
+- You have a [PostgreSQL instance](https://www.postgresql.org/download/). To improve data security, GX recommends using a separate user service account to connect to GX Cloud.
+
+- You know your PostgreSQL access credentials.
+
+### Connect to a PostgreSQL Data Asset 
+
+1. In GX Cloud, click **Data Assets** > **New Daata Asset**.
+
+2. Click the **New Data Source** tab and then select **PostgreSQL**.
+
+3. Enter a meaningful name for the Data Asset in the **Data Source name** field.
+
+4. Enter a connection string in the **Connection string** field. The connection string format is `postgresql://YourUserName:YourPassword@YourHostname:5432/YourDatabaseName`. 
```

Comment:
@Kilo59 @wookasz The connection string format should be `postgresql+psycopg2//YourUserName:YourPassword@YourHostname:5432/YourDatabaseName`